includes/class-sensei-admin.php

```diff
@@ -228,6 +228,8 @@ function install_pages_output() {
 	/**
 	 * create_page function.
 	 *
+	 * @deprecated since 4.0.0 use Sensei_onboarding\Pages::create_page
```

Comment:
Should it be `3.1.0`? Another thing is that we need to remember to add it to the issue to remove the deprecation code ;)

Reply:
Yup, fixed in b935750.